tests/refc/ccompilerArgs/library/Makefile

```
@@ -0,0 +1,10 @@
+all: libexternalc.so
+
+externalc.o: externalc.c externalc.h
+	cc -c -fPIC $< -o $@
```

Although it may never happen to be necessary, I suggest `$(CC)` here (and on line 7 below) for portability.
```suggestion
	$(CC) -c -fPIC $< -o $@
```

---

Agreed. I had something like this in a previous version of this patch, but had some difficulties with my setup. I'll give it another shot, as I agree that it is the way we *should* be doing it.

---

Also, it may be that this is the cause of the failing makefile on NIX & Ubuntu.